tests/sentry/integrations/utils/test_patch_parsers.py

```diff
@@ -0,0 +1,56 @@
+from sentry.integrations.utils.patch_parsers import PythonPatch
+from sentry.testutils.cases import TestCase
+
+
+class PythonPatchTestCase(TestCase):
```

Comment:
A test case for inputs that are not patches (e.g. malicious inputs) would be good to have.

Reply:
what would be a malicious input in this case? do you mean inputs that don't match the regex at all?

Reply:
@cathteng Yea, strings that won't match the regex is that I was thinking of.